mgo.go

```diff
@@ -330,6 +328,10 @@ func (inst *MgoInstance) run() error {
 		storageEngine := mongoStorageEngine(inst.EnableReplicaSet)
 		if storageEngine != "" {
 			mgoargs = append(mgoargs, "--storageEngine", storageEngine)
+			if storageEngine == "mmapv1" {
+				mgoargs = append(mgoargs, "--nssize", "1")
+				mgoargs = append(mgoargs, "--noprealloc", "--smallfiles")
```

Comment:
Nit: slightly weird that the first arg is on its own line (with parameter), but the next two are in a single append() call. More consistent if they were all in one append, or in 3 appends?